docs/source/smartdevice.rst

```diff
@@ -103,3 +115,9 @@ API documentation
 .. autoclass:: SmartDevice
     :members:
     :undoc-members:
+
+.. autoclass:: kasa.DeviceConfig
```

Comment:
Did you try what happens if you use just `DeviceConfig` without kasa here? I'm wondering if that's the cause why we are not seeing the class variable documentation.

Reply:
I see the class variable documentation in my generated html locally. Where are you looking that you don't see it?

Reply:
I'm looking at the readthedocs generated docs (by clicking the details link in the github action): https://python-kasa--614.org.readthedocs.build/en/614/smartdevice.html#kasa.DeviceConfig

Reply:
This is what I see when I visit https://python-kasa--614.org.readthedocs.build/en/614/smartdevice.html#kasa.DeviceConfig

![DeviceConfig](https://github.com/python-kasa/python-kasa/assets/51370195/8530d651-8de0-4cf3-82dc-21cfab49f9d4)

Reply:
Ohh, indeed... My mistake, sorry!